dev.lizardbyte.app.sunshine.yml

```diff
@@ -0,0 +1,215 @@
+app-id: dev.lizardbyte.app.sunshine
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+separate-locales: false
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: sunshine
+
+finish-args:
+  - --device=all
+  - --env=PULSE_PROP_media.category=Manager
+  - --persist=.config/sunshine
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
```

Comment:
fallback-x11?

Reply:
Needs XWayland for mouse position on the screen.